README.md

```diff
@@ -49,13 +55,12 @@ My breakdown of the operations I'd like.
 * set/get the default compiler (sets the link/script in PATH) (`zigup default` and `zigup default <version>`)
 * set/clear the "keep" flag on a compiler.  Each keep flag can also have a note explaining why it's being kept.
 * clean (cleans compilers without the "keep" flag and aren't the default)
+* keep a compiler (in conjunction with clean)
```

Comment:
Not sure whether this should be in the config or not.  We could say

* should we allow "keep" settings in the configuration file?